ext/src/helpers/file_stdin_stream.rs

```diff
@@ -0,0 +1,25 @@
+use std::sync::Arc;
+
+use wasmtime_wasi::{FileInputStream, StdinStream};
+
+pub struct FileStdinStream {
+    file: Arc<File>,
```

Comment:
I don't actually know what fields this struct _should_ have. I tried making a similar implementation to how `OutputFile` works but the types used by the constructor functions for are different.